austrakka/components/user/__init__.py

```diff
@@ -37,28 +41,32 @@ def user_list(show_disabled: bool, out_format: str):
 @opt_owner_group_roles(required=False)
 @opt_is_austrakka_process(default=False)
 def user_add(
-    user_id: str,
-    org: str,
-    owner_group_roles: List[str],
-    is_austrakka_process: bool,
+        user_id: str,
+        org: str,
+        owner_group_roles: List[str],
+        is_austrakka_process: bool,
 ):
```

Comment:
If not updating `user add` and the corresponding endpoint, should add a card for this for us to do soon